taxi/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path
+from django.conf import settings
+from django.conf.urls.static import static
+from . import views

```

Comment:
The import of `views` is unnecessary if you are not using it in this file. Consider removing it to clean up the code.